### PR TITLE
Sunblocker

### PR DESCRIPTION
**Pull request by Athanaseus**
- Add sunblocker cab.
- Test run:
```
    flag-solar-rfi:
      info: flag target solar-like RFI using sunblocker
      cab: sunblocker
      params:
        command: phazer
        inset: =recipe.ms
        outset: '<new.ms>' # don't set to write lags to current ms
        imsize: 1024
        cell: 1.0
        pol: 'i'
        threshmode: 'mad'
        threshold: 10
        mode: 'all'
        radrange: 0
        angle: 0
        show: ".sunblocker.svg"
        showdir: =recipe.dir-out-base
        verb: True
        dryrun: False
        uvmax: 2000
        uvmin: 0
```

### CODE REVIEW

File: cultcargo/builder/cargo-manifest.yml [hunk 1]
```diff
@@ -28,6 +28,13 @@ images:
     versions:
       base: {}
 
+  aimfast:
+    versions:
+      '1.3.4':
+        package: aimfast==1.3.4
+      base:
```
Comment (o-smirnov) [inline cultcargo/builder/cargo-manifest.yml:35]: Why not call it 'master' or 'main', for consistency with the other packages? (Meaning tracking the master or main branch).

File: cultcargo/sunblocker.yml [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+        info: A pyrap method to remove solar-like RFI from interferometric data (https://github.com/gigjozsa/sunblocker)
+        image:
+            _use: vars.cult-cargo.images
+            name: sunblocker
+        flavour:
+            kind: python-code
+            input_vars: false
+            input_dict: options
+        command: |
+            import subprocess
```
Comment (o-smirnov) [inline cultcargo/sunblocker.yml:15]: No need for import, since it's commented out.

File: cultcargo/sunblocker.yml [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+            from sunblocker.sunblocker import Sunblocker
+            from omegaconf import OmegaConf
```
Comment (o-smirnov) [inline cultcargo/sunblocker.yml:17]: Also seems unnecessary

File: cultcargo/sunblocker.yml [hunk 4]
```diff
@@ -0,0 +1,148 @@
...
+           
+            #subprocess.call("pip install matplotlib -U")
+            func = options.pop('command')
+            run_func = getattr(Sunblocker(), func, None)
+            if run_func is None:
+                raise RuntimeError(f"Function {function} is not part of Sunblocker()") 
+            for arg in options.keys():
+                if options[arg] in [None]:
```
Comment (o-smirnov) [inline cultcargo/sunblocker.yml:25]: Why not options[arg] is None, simpler? Or options = {key: value for key, value in options.items() if value is not None}, but I'm overly fond of list/dict comprehensions, I know. :)

File: cultcargo/builder/cargo-manifest.yml (hunk 1 above)
Comment (Athanaseus) [reply cultcargo/builder/cargo-manifest.yml]: duly

File: cultcargo/sunblocker.yml (hunk 2 above)
Comment (Athanaseus) [reply cultcargo/sunblocker.yml]: duly

File: cultcargo/sunblocker.yml (hunk 3 above)
Comment (Athanaseus) [reply cultcargo/sunblocker.yml]: duly

File: cultcargo/sunblocker.yml (hunk 4 above)
Comment (Athanaseus) [reply cultcargo/sunblocker.yml]: neat

**Review by o-smirnov — APPROVED** [review]
LGTM!

**Comment by o-smirnov** [comment]
As per https://github.com/caracal-pipeline/cult-cargo/discussions/10#discussioncomment-8485312: because this PR kicks off a new pre-release cycle (0.1.3), cc-0.1.3 images for all cabs need to be uploaded to quay.io before the build will pass. Please run ``build-cargo -a`` (in a tmux session on a big node) and then restart the checks here.

**Pull request merged**